src/images/generic.rs

```diff
@@ -55,7 +55,7 @@ impl GenericImage {
 }
 
 impl Image for GenericImage {
-    type Args = Vec<String>;
```

Comment:
If we do this, then we cannot pass any arguments for generic images. I think this has to stay!